src/EventStore.Core/LogV3/EventTypeIdToNameFromStandardIndex.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using EventStore.Core.Data;
+using EventStore.Core.LogAbstraction;
+using EventStore.Core.Services.Storage.ReaderIndex;
+using EventStore.Core.TransactionLog.LogRecords;
+
+namespace EventStore.Core.LogV3 {
+	public class EventTypeIdToNameFromStandardIndex : INameLookup<uint> {
```

Comment:
we are relying on LogV3SystemEventTypes.EventTypeInterval being 1 here
consider adding a test specifically for this class that will fail if the interval changes